docs/developer/issue-triage.md

```diff
@@ -0,0 +1,72 @@
+# Managing issues
+
+We value community engagement with Alloy and want to encourage community members and all users to contribute to Alloy's
+development by raising issues for bugs, feature requests, proposals, and more. To ensure that issues are appropriately
+reviewed and users get the feedback they desire we have a few patterns for issue triaging.
+
+## Issue triage process
+
+Issues that are created in the Alloy repository should all begin with the `needs-triage` label.
+They may also start with other labels like `type/docs`, `proposal`, or `bug` depending on the template selected.
+
+The goal of the triage process is to collect enough information to move the issue into the right category or categories
+where it can then be most easily tracked to completion. The triage process might include asking clarifying questions of the author,
+researching the behavior of Alloy or other technologies relevant to the issue, and reproducing an issue in a test environment.
+
+### Issue states and labels
+
+After an effort has been made to triage an issue, the issue should be in one of several states
+
+* Waiting for author
+  * An issue might be waiting on more feedback from its author if
+    * There was insuffucient information available to reproduce the issue
+    * There was insufficient information available to fully define the feature requested
+    * An answer to the author's problem was proposed using existing functionality
+  * These issues should be tagged `waiting-for-author` in addition to any other categorizing tags (`bug`, `enhancement`, etc)
+* Waiting for codeowner
+  * Some issues related to community components within Alloy will be dependent on the community maintainer of the component, and other issues may be dependent on codeowners of components that are not maintained by additional teams within Grafana (beyla, pyroscope, database_observability)
+  * These issues should be tagged `waiting-for-codeowner` in addition to any other triage that is initially possible
+  * These issues should retain the `needs-triage` label until the codeowner has responded
+* Ready to implement/fix/document
+  * An issue is ready to implement or fix if the scope is well understood, and if it is an issue it should be replicatable
+  * These issues should be tagged `bug`, `enhancement`, `type/docs`, or `flaky-test`
+  * If the issue should be in the next release, it should be tagged `release-blocker`
```

Comment:
> `release-blocker`

This would be cool to build automation around if we don't already have it.

Reply:
Definitely agree, I don't think we have it at the moment.